environments/mobile-development.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Mobile Development
+description: Learn how to develop mobile apps with Coder.
+---
+
+Mobile applications frequently require intensive workloads to achieve fast compilations.
+With Coder, you can save time compiling while getting a native emulator experience.
+
+## Android
+
+Use the Android emulator on your local machine to build and debug remote apps.
+
+1. On your local machine, install [Android Studio](https://developer.android.com/studio).
+
+   ```sh
+   brew install android-studio
+   ```
+
+1. Install the SDK.
+   ![Android SDK Install](../assets/android-sdk-missing.png)
+1. [Create and start a device.](https://developer.android.com/studio/run/managing-avds)
+   ![Android Device](../assets/android-avd.png)
+1. Set the installation path of your Android SDK to the environment variable `ANDROID_SDK_PATH`.
```

Comment:
Would be good to show how to get to this preferences page to find this value, maybe something like:
Preferences > System Settings > Android SDK